performance_test.lua

```diff
@@ -0,0 +1,74 @@
+local RNG_SEED = 69420731
```

Comment:
Here's a the benchmark I was using during development. Not sure if it makes sense to merge it in. Let me know if you want this removed or moved to a subfolder or something.